roles/thelounge/vars/main.yaml

```
@@ -0,0 +1,3 @@
+_thelounge__host: '{{ "127.0.0.1" if thelounge__apache_proxy | d(False) else
+  thelounge__host }}'
+_thelounge__reverse_proxy: '{{ thelounge__apache_proxy | d(False) }}'
```

I've been trying to standardize on _not_ use double underscores for private variables. It seems to be the dominating style in our repositories by an order of magnitude.

```suggestion
_thelounge_reverse_proxy: '{{ thelounge__apache_proxy | d(False) }}'
```

(I won't block approval on this, but in that case I'd like for you to explicitly nack it, if you really prefer this style. Thanks!)

---

Thanks, I removed it because I removed the apache proxy. I will take it in account in the future.